src/ln/channelmanager.rs

```diff
@@ -155,6 +155,23 @@ impl MsgHandleErrInternal {
 			needs_channel_force_close: true,
 		}
 	}
+
+	#[inline]
+	fn disconnect_peer_with_err_msg(err: &'static str, channel_id: [u8; 32]) -> Self {
```

Comment:
Can we keep the naming consistent and mention close/no-close in the fn name as thats the most important distinction (also in your use-case here you need no-close, cause a random third-party sent us a message for the wrong channel so we shouldn't close it on the actual peer's channel).